DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h

```diff
@@ -0,0 +1,189 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+
+#ifndef DataFormats_CTPPSReco_CTPPSPixelLocalTrack_H
+#define DataFormats_CTPPSReco_CTPPSPixelLocalTrack_H
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h"
+#include "DataFormats/CTPPSDetId/interface/CTPPSPixelDetId.h"
+
+
+#include "TVector3.h"
+#include "TMatrixD.h"
+#include "TVectorD.h"
```

Comment:
Please move 
- from TVector3 to math::GlobalPoint
- from TVectorD to math::Vector<dimension> 
- from TMatrixD to math::Error<dimension>

These are still ROOT classes, but do not carry TObject overhead and have compiled size specs